p2p/src/lib.rs

```diff
@@ -23,6 +23,7 @@ pub mod net;
 pub mod peer_manager;
 pub mod rpc;
 pub mod sync;
+pub mod testing_utils;
```

Comment:
Preferably, testing support like this would be in a separate crate or at least guarded behind a feature so it's not included in production code

Reply:
Right. As I explained above it's not possible to use this module from a separate crate. Feature flag seems to work. Thanks for the hint.